mama/c_cpp/src/c/subscription.c

```diff
@@ -658,8 +655,7 @@ mamaSubscription_setupBasic (
     }
 
     /*Create the DQ strategy - sequence number checking etc.*/
-    if (MAMA_STATUS_OK!=(status=dqStrategy_create (&self->mDqStrategy,
-                      subscription)))
+    if (MAMA_STATUS_OK!=(status=mamaPlugin_fireSubscriptionPostCreateHook (subscription)))
     {
         mama_log (MAMA_LOG_LEVEL_ERROR,
                   "Could not create DQ strategy. [%s]",
```

Comment:
This section is no longer responsible for creating dq so comment and logging should reflect / be moved. Also conscious that plugin only looks to preserve the *last* status code so if multiple hooks are registered, this error detection will not work.

Reply:
good call, that logging is in the hook in DQ strategy plugin, the plugin.c code handles the error code handling